docs/release-notes/july-2023.md

```diff
@@ -0,0 +1,215 @@
+- Dockerfile: Single-quoted strings are now parsed without an error. ([#7780](https://github.com/returntocorp/semgrep/pull/7780))
+- Julia: Fixed a bug where try-catch patterns would not match properly. Now, you can use an empty try-catch pattern, such as:
+    ```julia
+    try
+      ...
+    catch
+      ...
+    end
+    ```
+    to catch only Julia code which does not specify an identifier for the catch.
+    - Otherwise, if you want to match any kind of try-catch, you can specify an ellipsis for the catch identifier instead:
+    ```julia
+    try
+      ...
+    catch ...
+      ...
+    end
+    ```
+    and this matches any try-catch, including those that do not specify an identifier for the catch. It is strictly more general than the previous.
+- TypeScript and JavaScript: Fixed an issue leading to incorrect autofix results involving JS/TS async arrow functions, for example, `async () => {}` ([#7353](https://github.com/returntocorp/semgrep/pull/7353))
+- Go: Fixed issue with patterns such as:
+    - `make(...);` 
+    - `make(...,$X);` 
+    - `make($A,$B)` ([#8171](https://github.com/returntocorp/semgrep/pull/8171))
+- Rust: Fixed an issue where implicit returns did not allow taint to flow, and various other small translation issues that would affect taint. ([#8325](https://github.com/returntocorp/semgrep/pull/8325))
+- Rust: Fixed attribute patterns to allow matching on simple attribute syntax. ([#8234](https://github.com/returntocorp/semgrep/pull/8234))
+- Rust: Fixed a bug where standalone metavariable patterns were not matching as expected. ([#8206](https://github.com/returntocorp/semgrep/pull/8206)) 
+- Rust: Macro calls which involve dereferencing and reference operators (such as `foo!(&x)` and `foo!(*x)`) now properly transmit taint.
+- Fixed Python Semgrep pattern parsing to also parse match statements, by chaining in the Python tree-sitter parser, and adding metavariable support to the Python tree-sitter parser.
+- Aliengrep mode: Fix whitespace bug preventing correct matching of parentheses. ([#7990](https://github.com/returntocorp/semgrep/pull/7990))
+- Fixed stack overflow caused by symbolic propagation.
+
+
+### Removed
+
+- **Dart** has been removed from experimental support.
+
+## Semgrep Cloud Platform
+
+### Added
+
+- Jira integration is now in **private beta** for existing customers.
+    - You can **create Jira tickets** from Semgrep Cloud Platform by following steps in the [Jira integration documentation](/semgrep-cloud-platform/jira/).
+    - To enable this feature:
+         1. Fill out the following form: [Request access to the Semgrep Jira integration private beta](https://get.semgrep.dev/Jira-private-beta.html).
+        2. Contact your Technical Account Manager or your Account Executive and let them know you'd like to try Jira integration out.
+- Usage limits for Team tier users are now in effect as of July 31, 2023. See the [Usage limits FAQ](/usage-limits) to learn more.
```

Comment:
note: I'm not sure if we should say Team tier here. I think it's just true for all those above 10 contributors. cc @sachishah for language

Reply:
I kinda think we could even say that usage limits are in effect, blanket - they just don't do anything if you're 10 or below. But would defer to Sachi on language.

Reply:
Got it, let's keep it simple. Thank you!